test-app/build-tools/android-metadata-generator/src/src/com/telerik/metadata/Generator.java

```diff
@@ -19,33 +19,39 @@
     /**
      * @param args
      */
-    public static void main(String[] args) throws Exception {
-        String metadataOutputDir = "bin";
-        List<String> params = null;
-
+    public static void main(String[] args) {
         try {
-            metadataOutputDir = getFileRows(MDG_OUTPUT_DIR).get(0);
-        } catch (Exception e) {
-            throw new InvalidParameterException(String.format("You need to pass a file containing a single line: the output dir for the metadata generator1\n", e.getMessage()));
-        }
-        try {
-            params = getFileRows(MDG_JAVA_DEPENDENCIES);
-        } catch (Exception e) {
-            throw new InvalidParameterException(String.format("You need to pass a file containing a list of jar/class paths, so metadata can be generated for them!\n", e.getMessage()));
-        }
+            String metadataOutputDir;
+            List<String> params;
 
-        TreeNode root = Builder.build(params);
+            try {
+                metadataOutputDir = getFileRows(MDG_OUTPUT_DIR).get(0);
+            } catch (Exception e) {
```

Comment:
Perhaps it's better to catch `Throwable` instead of `Exception` as we will catch Java `Error`s as well (e.g. `OutOfMemoryError`) and show them colored as well.

Reply:
The same applies for the rest of the generators, too.